### ISSUE TITLE
package d3treeR not available

### ISSUE THREAD

**Issue by patcrao** [timelyportfolio/d3treeR#33]
hello
i would like to make interactive treemaps with d3treeR but i cannot find the package. Can anyone help me?

thank you very much

**Comment by timelyportfolio** [comment]
`d3treeR` is not on CRAN.  You can install using `devtools::install_github("timelyportfolio/d3treeR")`.  Let me know if this does not work.

**Comment by patcrao** [comment]
Yes thanks. That is right. I found the solution meanwhile. In addition to what you say, i had also to pass through the proxy, writing this instruction:
set_config(use_proxy(url="....",port=80, username="", password=))

So i can do interactive treemaps now!! It is nice.
But now i have a new problem, so if you don't mind, i will open a new issue.

Thank you very much again. And talk to you soon.

**Comment by cw-huiyan** [comment]
Hi,

I tried to install the package via the devtools (devtools::install_github("timelyportfolio/d3treeR")) but failed. The error message goes: 
```
> devtools::install_github("timelyportfolio/d3treeR")
Downloading GitHub repo timelyportfolio/d3treeR@master
from URL https://api.github.com/repos/timelyportfolio/d3treeR/zipball/master
Installing d3treeR
"C:/PROGRA~1/R/R-34~1.3/bin/x64/R" --no-site-file --no-environ --no-save --no-restore --quiet CMD  \
  INSTALL  \
  "C:/Users/Huiyan/AppData/Local/Temp/RtmpE30ILe/devtools2100650e6c1/timelyportfolio-d3treeR-28c32e0"  \
  --library="C:/Users/Huiyan/Documents/R/win-library/3.4" --install-tests 

* installing *source* package 'd3treeR' ...
** R
** inst
** preparing package for lazy loading
Error : object 'cividis' is not exported by 'namespace:viridisLite'
ERROR: lazy loading failed for package 'd3treeR'
* removing 'C:/Users/Huiyan/Documents/R/win-library/3.4/d3treeR'
In R CMD INSTALL
Installation failed: Command failed (1)
```

Can you please help? I would really love to be able to use your visualization. Looks amazing!

Thanks!

**Comment by timelyportfolio** [comment]
@cw-huiyan, great to hear and glad you are attempting to use.  I have not done much work on this lately.  The issue it appears is an unstated dependency on `viridisLite` in one of the dependencies.  Try

```
install.packages("viridisLite")
```

Please let me know if this resolves the problem.

**Comment by cw-huiyan** [comment]
@timelyportfolio It worked! Yay!!! Thank you so much! :D

**Comment by timelyportfolio** [comment]
@cw-huiyan great!  Please provide feedback as you use it.  Closing this issue, but please feel free to repopen.

**Comment by bac3917** [comment]
I installed the viridis package and then used devtools to grab d3treeR, but i get the following error:

The downloaded binary packages are in
	C:\Users\...\AppData\Local\Temp\RtmpiKQ7TN\downloaded_packages
"C:/PROGRA~1/R/R-34~1.2/bin/x64/R" --no-site-file --no-environ --no-save --no-restore --quiet  \
  CMD INSTALL  \
  "C:/Users/.../AppData/Local/Temp/RtmpiKQ7TN/devtools1f4011573d52/d3treeR-d3treeR-ebb833d"  \
  --library="C:/Program Files/R/R-3.4.2/library" --install-tests 

ERROR: dependency 'XML' is not available for package 'd3treeR'
* removing 'C:/Program Files/R/R-3.4.2/library/d3treeR'
Installation failed: Command failed (1)

**Comment by sada1993** [comment]
@bac3917 XML is one of the Imports for d3treeR, do you have it installed in your R environment?

`install.packages("XML")`

**Comment by bac3917** [comment]
@sada1993   I installed XML but still get the following Warning and Errors:

    package ‘XML’ successfully unpacked and MD5 sums checked

    The downloaded binary packages are in
    	C:\Users\bcohen\AppData\Local\Temp\Rtmp0qAByU\downloaded_packages
    install.packages("d3treeR")
    Warning in install.packages :
    package ‘d3treeR’ is not available (for R version 3.5.1)
    library(d3treeR)
    Error in library(d3treeR) : there is no package called ‘d3treeR’

    sessionInfo()
    R version 3.5.1 (2018-07-02)
    Platform: x86_64-w64-mingw32/x64 (64-bit)
    Running under: Windows >= 8 x64 (build 9200)

**Comment by bac3917** [comment]
Also @sada1993 , I tried an install with `devtools`:

    install_github("sada1993/d3treeR")
    Downloading GitHub repo sada1993/d3treeR@master
    from URL https://api.github.com/repos/sada1993/d3treeR/zipball/master
    Installing d3treeR
    "C:/PROGRA~1/R/R-35~1.1/bin/x64/R" --no-site-file --no-environ --no-save --no-restore --quiet CMD     INSTALL  \
    "C:/Users/bcohen/AppData/Local/Temp/Rtmp0qAByU/devtools25a445636e00/sada1993-d3treeR-01f22d9"  \
      --library="C:/Program Files/R/R-3.5.1/library" --install-tests 

    * installing *source* package 'd3treeR' ...
    ** R
    ** inst
    ** byte-compile and prepare package for lazy loading
    Error : package 'influenceR' was installed by an R version with different internals; it needs to be     reinstalled for use with this R version
    ERROR: lazy loading failed for package 'd3treeR'
    * removing 'C:/Program Files/R/R-3.5.1/library/d3treeR'
    In R CMD INSTALL
    Installation failed: Command failed (1)
